doc/user/isisd.rst

```diff
@@ -79,6 +79,10 @@ writing, *isisd* does not support multiple ISIS processes.
    - wide
      Use new style of TLVs to carry wider metric. FRR uses this as a default value
 
+.. clicmd:: advertise-high-metrics
```

Comment:
May be a short description about in which context the paramater should be used and which metric value is sent

Reply:
Updated